src/queries/index.js

```diff
@@ -107,7 +107,11 @@ export const createUser = ({
   image,
   firebaseID,
   module,
-  program
+  program,
```

Comment:
Nice job updating the createUser query to take in the additional info we are gathering from each user. I think the skills will really help earlier mods when they are looking for upper mod students to pair with